firebase-functions/functions/src/firebase/calculateResult.ts

```diff
@@ -34,3 +43,8 @@ async function getCurrentRoomId(playerId: string): Promise<string> {
   const currentPlayer = await admin.firestore().collection(collections.players).doc(playerId).get();
   return currentPlayer.data()?.room;
 }
+
+async function getNumberOfPlayers(roomId: string): Promise<number> {
+  return (await admin.firestore().collection(collections.players).where('room', '==', roomId).get())
```

Comment:
Could you extract this magic string (and mine from line 38) to something similar to our `firebase-constants` of the angular project please?

Reply:
done!